build/dependencies.props

```diff
@@ -0,0 +1,195 @@
+<Project>
+
+  <ItemDefinitionGroup>
+    <ExternalDependency>
+      <!-- The NuGet package version. Floating versions not allowed. -->
+      <Version></Version>
+      <!-- Where does this pacakge come from -->
+      <Source></Source>
+      <!-- A list of warnings to suppress. -->
+      <NoWarn></NoWarn>
+      <!-- This dependency is 'Private', aka. it should not end up as a public-facing external dependency. This is validated by checking the nuspec on generated packages. -->
+      <Private>false</Private>
+    </ExternalDependency>
+  </ItemDefinitionGroup>
+
+  <!-- ASP.NET Core Tools feed -->
+  <PropertyGroup>
+    <RoslynFeed>https://dotnet.myget.org/F/roslyn/api/v3/index.json</RoslynFeed>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ExternalDependency Include="Microsoft.CodeAnalysis.Common" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)">
+      <!-- This version is required for Razor to build the VSIX for VS. -->
+      <NoWarn>KRB2004</NoWarn>
+      <Private>true</Private>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.CodeAnalysis.CSharp" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)" >
+      <!-- This version is required for Razor to build the VSIX for VS. -->
+      <NoWarn>KRB2004</NoWarn>
+      <Private>true</Private>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.CodeAnalysis.EditorFeatures.Text" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)">
+      <Private>true</Private>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.CodeAnalysis.Remote.Razor.ServiceHub" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)">
+      <Private>true</Private>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)" >
+      <!-- This version is required for Razor to build the VSIX for VS. -->
+      <NoWarn>KRB2004</NoWarn>
+      <Private>true</Private>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)">
+      <Private>true</Private>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.VisualStudio.LanguageServices" Version="2.6.0-beta1-61924-08" Source="$(RoslynFeed)">
+      <Private>true</Private>
```

Comment:
Some items in this file set `Private` as a child element while others set it inline.  Should we be consistent?

Reply:
yeah. I'll update